test/spawn_ecsact_cli.cc

```diff
@@ -0,0 +1,17 @@
+#include "spawn_ecsact_cli.hh"
+
+#include <boost/process.hpp>
+
+namespace bp = boost::process;
+
+int spawn_ecsact_cli(std::string executable, std::vector<std::string> args) {
```

Comment:
this function name could be better. its really just a simple spawn since you pass in the executable as a parameter 😆